### PR TITLE
increase coverage of contracts package

### LINKED ISSUE #2208: Increase code coverage for contracts package
The code coverage for the contracts package on `develop` branch is 90.49%. We should increase that as much as possible to 100%.

### PR DESCRIPTION
**Pull request by tonykogias**
**Metadata**
- Fixes #2208

### CODE REVIEW

**Comment by tonykogias** [comment]
@smartcontracts I managed to increase the statement coverage from `90.95%` to `99.18 %`. As I was on it I tried to increase the branch coverage as much as possible too. I managed to raise it from `84.44%`  to `93.38%` but I noticed that some of the `require`s are impossible to hit. If you have any idea on how to hit the missing statements/branches please let me know and I will try to work it out.

Also, I noticed that you are still using smock's old version. Is migrating from smock v1 to v2 something you are thinking of doing in the future? If so let me know and I can help since I got the hang of it the past few days.

**Comment by smartcontracts** [comment]
@tonykogias this is amazing work, thank you. I have been on break this past week but I will make sure to review this first thing next week! Upgrading to smock v2 is definitely important for us in the future (smock v1 relies on some old dependencies that I'd like to remove).

File: packages/contracts/test/contracts/L1/deployment/AddressDictator.spec.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  describe('getNamedAddresses', () => {
+    it('should return all the addresses and their names', async () => {
+      expect(await AddressDictator.getNamedAddresses()).to.be.deep.equal([
+        ['addr1', NON_ZERO_ADDRESS],
+      ])
+    })
+  })
+
+  describe('returnOwnership', () => {
+    it('should transfer contractc ownership to finalOwner', async () => {
```
Comment (smartcontracts) [inline packages/contracts/test/contracts/L1/deployment/AddressDictator.spec.ts:79]: Minor typo here

File: packages/contracts/test/contracts/L1/messaging/L1StandardBridge.spec.ts [hunk 2]
```diff
@@ -564,4 +575,10 @@ describe('L1StandardBridge', () => {
       )
     })
   })
+
+  describe('donateETH', () => {
+    it('it should just call the function', async () => {
+      await expect(L1StandardBridge.donateETH()).to.not.be.reverted
```
Comment (smartcontracts) [inline packages/contracts/test/contracts/L1/messaging/L1StandardBridge.spec.ts:581]: Could also confirm here that when ETH value is sent alongside the tx, it doesn't revert.

**Review by smartcontracts — APPROVED** [review]
This is really excellent work, thank you @tonykogias. Left some minor nitpicks, but otherwise looks really good.

**Comment by smartcontracts** [comment]
Let's get this combined into a few clean commits using the conventional commit format and we should be good to go. I don't think we need a changeset because we're only touching test code here.

**Review by mslipper — APPROVED** [review]

**Pull request merged**